templates/cli-start/.actor/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Specify the base Docker image. You can read more about
+# the available images at https://docs.apify.com/sdk/js/docs/guides/docker-images
+# You can also use any other image from Docker Hub.
+FROM apify/actor-node:22
+
+# Install system dependencies
+RUN apk add --no-cache jq wget bash
+
+# Install the Apify CLI
+RUN npm --quiet set progress=false \
+    && npm install -g apify-cli \
+    && echo "Installed NPM packages:" \
+    && (npm list --omit=dev --all || true) \
+    && echo "Node.js version:" \
+    && node --version \
+    && echo "NPM version:" \
+    && npm --version \
+    && rm -r ~/.npm
```

Comment:
don't you need `-f` too for folders?

Reply:
Huh. Truth be told, this is just copypasted. I doubt it'd work on nonempty folders, so I might as well just remove it.